Formula/arangodb.rb

```diff
@@ -21,16 +21,28 @@ class Arangodb < Formula
     cause "Fails with compile errors"
   end
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
 
+  resource "starter" do
+    url "https://github.com/arangodb-helper/arangodb.git",
+        :revision => "38953b049a38c844ffdbf667e55e60c6cfa22e70"
```

Comment:
We need a comment above stating what it is. And some versioning system, we prefer (as a rule) building from archives (`
https://github.com/arangodb-helper/arangodb/archive/0.13.10.tar.gz`)

Reply:
I took riak.rb as an example. Will change.